macros/audio.asm

```diff
@@ -215,7 +215,7 @@ ENDM
 	const unknownmusic0xee_cmd ; $ee
 unknownmusic0xee: MACRO
 	db unknownmusic0xee_cmd
-	dn \1, \2 ; left output enable mask, right output enable mask
+	db \1, \2 ; address
```

Comment:
Since the 2 bytes are a 16-bit address, they should be a single parameter. And the addresses used in the two songs can be replaced with labels, like I already explained `$4546` is `Music_Route1_Ch2`.